### ISSUE TITLE
can hallo drive real video

### ISSUE THREAD

**Issue by ljiqy** [fudan-generative-vision/hallo#141]
can hallo drive real video?Have the people in the video say certain things?

**Comment by xumingw** [comment]
Hallo drive image with audio, so people in the video say what you provide

**Comment by subazinga** [comment]
Do you mean the way like Lip Sync of Runway? I think this a good topic to follow. But currently our method only supports audio-driven feature.

**Comment by ljiqy** [comment]
> Do you mean the way like Lip Sync of Runway? I think this a good topic to follow. But currently our method only supports audio-driven feature.

yes, I mean I input a video with a person and an audio, can the video say the audio?
